groups/iam/module-iam-policies/main.tf

```diff
@@ -0,0 +1,27 @@
+# Create policies
+resource "aws_iam_policy" "iam_policy_plan" {
+  name        = "test-data-generator-stack-terraform-plan"
+  description = var.policy_description
+  policy      = data.aws_iam_policy_document.test_data_generator_stack_terraform_plan.json
+}
+
+resource "aws_iam_policy" "iam_policy_apply" {
+  name        = "test-data-generator-stack-terraform-apply"
+  description = var.policy_description
+  policy      = data.aws_iam_policy_document.test_data_generator_stack_terraform_apply.json
+}
+
+resource "aws_iam_policy" "iam_policy_destroy" {
+  name        = "test-data-generator-stack-terraform-destroy"
+  description = var.policy_description
+  policy      = data.aws_iam_policy_document.test_data_generator_stack_terraform_destroy.json
+}
+
+# Outputs
+output "iam_policies_arns" {
```

Comment:
I would put all the outputs into a dedicated `outputs.tf` file. I don't know if I've updated the style guide yet but it's something we want to adopt going forwards. This way all the outputs are in one place and are easy to find

Reply:
Done. Yeah, I did think about doing this - but as the 3 output were so closely associated with the three resources in main.tf, I'd originally decided to keep them together. But I see the advantage with sticking to a standard convention - so now updated. 👍